app/views/merchants/show.html.erb

```diff
@@ -0,0 +1,113 @@
+      <% elsif item.order.status == "paid"%>
+        <% cart_total = product.price * item.quantity %>
+        <%@paid_revenue += cart_total%>
+      <% elsif item.order.status == "complete"%>
+        <% cart_total = product.price * item.quantity %>
+        <%@complete_revenue += cart_total%>
+      <% elsif item.order.status == "cancelled"%>
+        <% cart_total = product.price * item.quantity %>
+        <%@cancelled_revenue += cart_total%>
+      <% end %>
+    <% end %>
+  <% end %> </p>
+
+<div class="row">
+  <div class="col-md-8 col-md-offset-2">
+    <table class="table table-bordered order-table" >
+      <thead>
+        <tr>
+          <th> Order Number </th>
+          <th> Items </th>
+          <th><div class="dropdown">
+            <button class="merchant-dropdown" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+              Order Status
+              <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+              <li><%= link_to "All", merchant_path(sort:'all')%></li>
+              <li><%= link_to "Pending", merchant_path(sort:'pending')%></li>
+              <li><%= link_to "Paid", merchant_path(sort:'paid')%></li>
+              <li><%= link_to "Complete", merchant_path(sort:'complete')%></li>
+              <li><%= link_to "Cancelled", merchant_path(sort:'cancelled')%></li>
+            </ul>
+          </div></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <% @orders.each do |order| %>
+            <td><%= link_to order.id, order %><br><small> Ordered on: <%= order.order_time.strftime("%m/%d/%Y %I:%M%p") if !order.order_time.nil? %></small></td>
+              <td>
+                <% order.products.each do |product| %>
+                  <% if product.merchant_id == @merchant.id%>
+                    <%= link_to product.name, product %>
+                    $<%= number_with_precision(product.price, :precision => 2, :delimiter => ',') %>
+                    <% product.order_items.each do |item|%>
+                      <% if item.order.id == order.id%><br>
+                        <%= "   Quantity: " + item.quantity.to_s %>
+                        <%= "   Total: $" + (item.quantity * product.price).to_s + "0" %>
+                          <%if item.shipped?%>
+                          <%= link_to("", :class => "gray-button btn btn-default") do %>
+                          <!-- This button goes nowhere, it just is a visual contrast to the "mark as shipped button" -->
+                            Item has shipped
+                            <% end %>
+                          <% elsif item.order.status == "paid" %>
+                            <%= link_to(shipped_order_item_path(item), :method => :put, :class => "btn btn-default", :id => 'all-button') do %>
+                            Mark As Shipped
+                          <% end %>
```

Comment:
This trail of `<% end %>` is perrrrrrty smelly guys. Feels like all this logic should be in your controller instead. (: